test/python/picongpu/quick/pypicongpu/grid.py

```diff
@@ -55,6 +56,24 @@ def test_types(self):
             g.boundary_condition_y = 1
         with self.assertRaises(TypeError):
             g.boundary_condition_z = {}
+        with self.assertRaises(TypeError):
+            g.n_gpus = [1, 1, 1] # list not accepted - tuple needed
+
+    def test_positiv(self):
```

Comment:
name: what is positive? -> `test_gpu_cnt_positive`
docstring: missing

Reply:
will add doc strings to all other tests as well